xExtension-YouTube/extension.php

```diff
@@ -69,16 +69,22 @@ public function loadConfigValues()
             return;
         }
 
-        if (!empty(FreshRSS_Context::$user_conf->yt_player_width)) {
+        if (FreshRSS_Context::$user_conf->yt_player_width != '') {
```

Comment:
You're reintroducing [a bug](https://github.com/kevinpapst/freshrss-youtube/pull/13) when user hasn't configured the extension yet. Alternative can be:

```php
if (isset(FreshRSS_Context::$user_conf->yt_player_width) && FreshRSS_Context::$user_conf->yt_player_width != '') {
    …
}
```

You'll have to take care of that too for the option you're adding.

Reply:
I just tried, but `isset` is always false on my case, even when there is actual values (probably related to the `empty` which was always true, which is why I removed them), so it force values to be on default… :/

Any ideas why ?

Reply:
I'll take a look after lunch ;)

Reply:
I can't reproduce the error I had the other time, really strange :disappointed: so let's keep it like this and sorry for the noise!

Reply:
Perhaps some updates in PHP ? Or changes in the way FreshRSS expose those contexts ?

Reply:
Nope I don't think so. It's more probably I was doing some other things and the errors were caused by my own code :thinking: